recipes-samples/images/lmp-image-common.inc

```diff
@@ -23,6 +23,7 @@ REQUIRED_DISTRO_FEATURES = "pam systemd"
 
 # Base packages
 CORE_IMAGE_BASE_INSTALL += " \
+    aktualizr-get \
```

Comment:
curious why this was here, but not also aktualizr-lite?

Reply:
Just because aktualizr-get is just a tool, while aktualizr-lite would end up replacing the functionality provided by aktualizr itself. The way I see we should have either aktualizr or aktualizr-lite, not both, right?

And I'm now checking if this can be set by the SOTA_CLIENT variable.